src/logic/community/broadcaster.ts

```diff
@@ -222,6 +242,85 @@ export function createCommunityBroadcasterComponent(
     )
   }
 
+  /**
+   * Gets online community members with pagination support
+   * @param {string} communityId - The ID of the community
+   * @param {string[]} onlineUsers - Array of online user addresses to filter by
+   * @param {string[]} excludedAddresses - Optional array of addresses to exclude
+   * @returns {Promise<string[]>} Array of online member addresses
+   */
+  async function getOnlineCommunityMembersAddresses(
+    communityId: string,
+    onlineUsers: string[],
+    excludedAddresses?: string[]
+  ): Promise<string[]> {
+    const onlineMemberAddresses: string[] = []
+    let offset = 0
+    let hasMore = true
+
+    while (hasMore) {
+      const batch = await communitiesDb.getCommunityMembers(communityId, {
+        pagination: { limit: MEMBER_FETCH_BATCH_SIZE, offset },
+        filterByMembers: onlineUsers,
+        excludedAddresses
+      })
+
+      if (batch.length === 0) break
+
+      batch.forEach(({ memberAddress }) => {
+        onlineMemberAddresses.push(memberAddress)
+      })
+
+      offset += MEMBER_FETCH_BATCH_SIZE
+      hasMore = batch.length === MEMBER_FETCH_BATCH_SIZE
+    }
+
+    return onlineMemberAddresses
+  }
+
+  /**
+   * Broadcasts voice chat started event to online community members
+   * @param {BroadcastableEvent} event - The event to broadcast
+   * @param {BroadcastOptions} options - Optional broadcast options (e.g., excludeAddresses)
+   */
+  async function broadcastVoiceChatStarted(event: BroadcastableEvent, options?: BroadcastOptions): Promise<void> {
+    const { metadata } = event as CommunityVoiceChatStartedEventReducedMetadata
+    const { communityId } = metadata
+
+    const onlineSubscribers = subscribersContext.getSubscribersAddresses()
+
+    if (onlineSubscribers.length === 0) {
+      return
+    }
+
+    // Get online members from the community, excluding specified addresses at database level
+    const excludedAddresses = options?.excludeAddresses?.map((addr) => addr.toLowerCase())
+    const onlineMemberAddresses = await getOnlineCommunityMembersAddresses(
```

Comment:
NIT:
This function, in reality, only grabs members based on address, right? Could we reuse the get members function we already have in the database and remove this one? maybe we can enhance the original one to expect options to exclude addresses, or we could exclude the address before calling the function